_episodes/03-computing-pi.md

```diff
@@ -87,6 +97,9 @@ We can demonstrate that this is much faster than the 'naive' implementation. Thi
 > - monolithic approach, less composable?
 {: .discussion}
 
+FIXME: Before this, the student has not learned about dask at all. It could
+be that students will struggle with this excercise, while the
+solution is actually very simple. It might be better to just show it?
```

Comment:
We actually show the first `dask` examples in the benchmarking chapter. It should be explained there that `dask.array` mimics the `numpy` API. We could repeat that information here as a hint to the exercise.

Reply:
I've added this to the FIXME.